### PR TITLE
mgr/dashboard: add columns to Pools table

### PR DESCRIPTION
**Pull request by alfonsomthd**
* Added columns:
Usage, Read bytes, Write bytes, Read ops, Write ops.

* TableKeyValueComponent:
Bugfix in method: _insertFlattenObjects

* Updated jest global mock:
Now window.getComputedStyle returns an object
that has getPropertyValue defined
(otherwise jest pool tests involving chart.js threw error). See:
https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle

Fixes: https://tracker.ceph.com/issues/37717
Fixes: https://tracker.ceph.com/issues/34320

Signed-off-by: Alfonso Martínez <almartin@redhat.com>

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by alfonsomthd** [comment]
jenkins retest this please

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts [hunk 1]
```diff
@@ -161,9 +191,24 @@ export class PoolListComponent implements OnInit {
...
     _.map(pools, (pool: object) => {
-      delete pool['stats'];
       pool['pg_status'] = this.transformPgStatus(pool['pg_status']);
+      const stats = {};
+      _.forEach(requiredStats, (stat) => {
+        stats[stat] = pool['stats'] && pool['stats'][stat] ? pool['stats'][stat] : emptyStat;
+      });
+      pool['stats'] = stats;
+
+      pool['stats']['rd_bytes']['series'] = pool['stats']['rd_bytes']['series'].map(
```
[the remark this answers is not in the document]
Comment (alfonsomthd) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts]: Done.

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts [hunk 2]
```diff
@@ -116,11 +123,34 @@ export class PoolListComponent implements OnInit {
...
+      },
+      {
+        prop: 'stats.wr_bytes.series',
+        name: this.i18n('Write bytes'),
+        cellTransformation: CellTemplate.sparkline
+      },
+      {
+        prop: 'stats.rd.rate',
+        name: this.i18n('Read ops'),
+        flexGrow: 1.1,
```
[the remark this answers is not in the document]
Comment (alfonsomthd) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts]: Done.

**Comment by callithea** [comment]
QA run was successful: http://pulpito.ceph.com/laura-2019-01-09_16:14:05-rados:mgr-wip-lpaduano-testing-25791-distro-basic-smithi/

**Pull request merged**